### ISSUE TITLE
Cors exception + Readme CopyPaste bug

### ISSUE THREAD

**Issue by lemonov** [wbhob/nest-middlewares#23]
```typescript
CorsMiddleware.configure({})` 
```
This throws unhandled promise rejection exception.

Passing null instead seems to do the trick
```typescript
CorsMiddleware.configure(null)` 
```

Also the Readme in CorsMiddleware uses HelmetMiddleware instead of CorsMiddleware. I assume this is a minor bug.

Is the `MiddlewaresConsumer` any different that `MiddlewareConsumer` (without **s**). Cause I couldnt find such class.

**Comment by wbhob** [comment]
Can you paste the call stack/promise rejection message? I can try to take a look this evening (because empty obj should work). Also, if you don't need to configure CORS, then you shouldn't be calling `.configure()` at all – it uses the default config by default.

**Comment by anodynos** [comment]
I confirm that with `nestjs@6.3.1`, using `CorsMiddleware.configure({});` we get 

```
(node:117) UnhandledPromiseRejectionWarning: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:470:11)
    at ServerResponse.header (/srv/node_modules/express/lib/response.js:767:10)
    at ServerResponse.send (/srv/node_modules/express/lib/response.js:170:12)
    at ServerResponse.json (/srv/node_modules/express/lib/response.js:267:15)
    at ExpressAdapter.reply (/srv/node_modules/@nestjs/platform-express/adapters/express-adapter.js:23:57)
    at ExceptionsHandler.catch (/srv/node_modules/@nestjs/core/exceptions/base-exception-filter.js:28:28)
    at ExceptionsHandler.next (/srv/node_modules/@nestjs/core/exceptions/exceptions-handler.js:15:20)
    at /srv/node_modules/@nestjs/core/router/router-proxy.js:12:35
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:117) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:117) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Omitting `CorsMiddleware.configure({});` or passing `null` solves the problem.
The same code used to work fine with NestJs@5.7 & CorsMiddleware@^5

**Comment by wbhob** [comment]
> Also, if you don't need to configure CORS, then you shouldn't be calling .configure() at all – it uses the default config by default.

**Comment by kctang** [comment]
Hi @wbhob & @anodynos ,

Took me two hours to realise this - I think the fix from #20 is not released in v6.0.0. https://github.com/wbhob/nest-middlewares/blob/v6.0.0/packages/cors/index.ts.

`cors()` is called without `else`.

With the fix, custom CORS options will work (i tested locally by making a copy of the middleware from master).

Hope you can release a new version soon so I don't have to dup a local copy of the CORS middleware.

**Comment by wbhob** [comment]
published v6.0.1
